src/Transport/AzureServiceBusTransport.cs

```diff
@@ -107,6 +107,14 @@ public class AzureServiceBusTransport : TransportDefinition
                     .ToArray();
 
                 await queueCreator.CreateQueues(allQueues, cancellationToken).ConfigureAwait(false);
+
+                foreach (IMessageReceiver messageReceiver in infrastructure.Receivers.Values)
```

Comment:
Given that we currently also don't run the other subscription creations concurrently and we are doing a patch release I'm :+1: to leave this as is